### ISSUE TITLE
BooleanField doesn't work properly with values_list, while using SQLite

### ISSUE THREAD

**Issue by opengs** [tortoise/tortoise-orm#325]
**Describe the bug**
BooleanField isnt converted from SQLite INT to python bool.
Looks like values_list command doesnt convert SQLite int to python bool.

**To Reproduce**
import asyncio
from tortoise import Tortoise
from tortoise.models import Model
from tortoise import fields

class Tournament(Model):
    id = fields.IntField(pk=True)
    bool_field = fields.BooleanField(default=True)

async def init():
    await Tortoise.init(
        db_url='sqlite://:memory:',
        modules={'models': ['__main__']}
    )
    await Tortoise.generate_schemas()
    await Tournament.create()
    result = await Tournament.filter(bool_field=True).values_list("bool_field")
    print(result)
    await Tortoise.close_connections()

loop = asyncio.get_event_loop()
loop.run_until_complete(init())

**Expected behavior**
After executing filter and values_list on the BooleanField, result contains bool value, not int.

**Comment by grigi** [comment]
I'm sure we fixed this same issue before. I suppose the tests didn't cover it properly.

**Comment by grigi** [comment]
:sob: This is valid apparently:
```py
self.assertEqual(1, True)
self.assertTrue(1)
```
Should be changed to:
```py
self.assertIs(1, True)
```

**Comment by grigi** [comment]
Should be fixed in 0.16.2 & 0.15.20. Please test.
